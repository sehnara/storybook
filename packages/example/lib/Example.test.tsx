import React from 'react';
import Example from './Example';

import renderer from 'react-test-renderer';

test('Example Renders', () => {
  const example = <Example text="test" />;
  const tree = renderer.create(example).toJSON();
  expect(tree).toMatchSnapshot();
});
