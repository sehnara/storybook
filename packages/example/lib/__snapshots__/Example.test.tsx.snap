// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example Renders 1`] = `
<View>
  <Text>
    test
  </Text>
</View>
`;
